operators/BaseOperator.js

```diff
@@ -127,7 +127,8 @@ class BaseOperator {
           });
       } else {
         processingLockStatus.conflict = true;
-        logger.debug(`Resource ${objectBody.metadata.name}  ${resourceGroup}, ${resourceType}, ${queryString} - is picked by process with ip ${lockedByManager} at ${processingStartedAt}`);
+        logger.debug(`Resource ${objectBody.metadata.name}  ${resourceGroup}, ${resourceType}, ${queryString} -` +
+          ` is picked by process with ip ${lockedByManager} at ${processingStartedAt} will be held for another ${elapsedTimeInMs}(ms)`);
```

Comment:
Shouldn't it be => Will be held for another `elapsedTimeInMs  - CONST.PROCESSING_REQUEST_BY_MANAGER_TIMEOUT`?